app/services/geek_jokes_api_service.js

```diff
@@ -0,0 +1,11 @@
+const axios = require('axios');
+
+const base_api_url = `${process.env.BASE_API_URL_GEEK_JOKES}/api?format=json`;
+
+exports.geekJokesApi = {
+  response: () =>
+    axios
+      .get(base_api_url)
+      .then(res => res.data.joke)
+      .catch(err => err)
```

Comment:
I recommend you to create a specific error and return this error. The errors are created in **_app/errors.js_** for example
`
exports.JOKE_ERRORS = 'default_error';
exports.jokeErrors= message => internalError(message, exports.JOKE_ERRORS);
`

In _**app/middlewares/errors.js**_ you should defined the status code error for example 

`
const statusCodes = {
  [errors.DATABASE_ERROR]: 503,
  [errors.DEFAULT_ERROR]: 500,
  [errors.JOKE_ERRORS]: 500
};
`